src/material-examples/dialog-content/dialog-content-example.ts

```diff
@@ -0,0 +1,26 @@
+import {Component} from '@angular/core';
+import {MdDialog} from '@angular/material';
+
+/**
+ * @title Dialog Content
+ */
+@Component({
```

Comment:
"Dialog content" by itself doesn't really communicate what this example is for. It should be something like 
"Dialog with header, scrollable content, and actions". The content should be long enough to cause scrolling and wide enough that the actions are aligned to either the right or left of the dialog.